graphdb/janus/src/main/java/org/apache/atlas/repository/graphdb/janus/AtlasEncodingUtil.java

```diff
@@ -0,0 +1,16 @@
+package org.apache.atlas.repository.graphdb.janus;
+
+import org.janusgraph.util.encoding.LongEncoding;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Objects;
+
+public class AtlasEncodingUtil {
```

Comment:
- Move the class under `utils` package
- Rename the class to `AtlasUtils` or simply `Utils` with method name as `toLongEncoding()`
- nit: Add `@javax.annotation.Nonnull` annotation in the params `toLongEncoding(@Nonnull Object obj)`.

Reply:
Cannot move to utils package. This class has JanusGraph dependency.